src/lib/y2storage/filesystems/btrfs.rb

```diff
@@ -301,6 +301,18 @@ def restore_unshadowed_subvolumes(devicegraph)
         end
       end
 
+      # Determines the default subvolume name
+      #
+      # When a default subvolume name have been used, a subvolume named after
+      # it lives under the #top_level_btrfs_subvolume. Otherwise, an empty
+      # string will be taken as the default subvolume name.
+      #
+      # @return [String] Default subvolume name
+      def default_subvolume_name
```

Comment:
I am not sure if this is the best name for the method. We have a `Btrfs#default_btrfs_subvolume` method, and `#default_subvolume_name` looks like the name (path) of that subvolume, but it is not. Probably we need a new subvolume concept, something like `top_parent_subvolume`. Something to discuss in future, I guess.

Reply:
Yes, the name can be tricky. For instance, this concept is named `btrfs_default_subvolume` in the control file, which is even worst: https://github.com/yast/yast-storage-ng/blob/master/test/data/control_files/caasp.xml#L31

Reply:
The problem when using `top` is that the `top_level` is a different thing and it could be confusing too.